### DISCUSSION TITLE
Cannot connect to Sonarr anymore

### DISCUSSION THREAD

**Discussion by mrmuiz** [sct/overseerr#3143] [Feature Requests]
Sonarr Version
    4.0.0.202
Requires a login. Now Overseerr wont connect to Sonarr anymore.
It is missing the option for a name and password.

**Comment by sideh** [comment]
From my limited knowledge you don't need to have the username/password. The API is all that is needed.

In Sonar go to: Settings > General > Security and Click the copy button on the API then,

In Overseerr go Settings > Services > then paste it in the API and test to see if it can connect.

Works for me at least.

**Reply by mrmuiz** [reply]
A few verssions later now. Its working again.
